src/apps/dashboard/routes/users/parentalcontrol.tsx

```diff
@@ -163,15 +164,15 @@ const UserParentalControl = () => {
         populateRatings(allParentalRatings);
 
         let ratingValue = '';
-        if (user.Policy?.MaxParentalRating) {
+        if (user.Policy?.MaxParentalRating != null) {
             allParentalRatings.forEach(rating => {
-                if (rating.Value && user.Policy?.MaxParentalRating && user.Policy.MaxParentalRating >= rating.Value) {
+                if (rating.Value != null && user.Policy?.MaxParentalRating != null && user.Policy.MaxParentalRating >= rating.Value) {
```

Comment:
Isn't the second `user.Policy?.MaxParentalRating != null` unnecessary? We have checked it above.

Reply:
Yes, but eslint still requires it. Not sure why.